src/Encoder/JsonEncoder.php

```diff
@@ -12,7 +12,7 @@ class JsonEncoder extends SymfonyJsonEncoder
     /**
      * {@inheritdoc}
      */
-    public function decode($data, $format, array $context = [])
+    public function decode($data, $format, array $context = []): mixed
```

Comment:
This makes it non 7.4 compatible.

Reply:
I think it's reasonable to just bump the major on the library.